### PR TITLE
Add privacy manifest to iOS plugin code

### LINKED ISSUE #2189: Add privacy manifest for Apple
### Use case

It's seems that patrol is requiring a privacy manifest.
See https://github.com/flutter/flutter/issues/131940 for context.

Apple announcement here: [developer.apple.com/support/third-party-SDK-requirements](https://developer.apple.com/support/third-party-SDK-requirements/)

May 1st deadline announcement here: [developer.apple.com/news/?id=3d8a9yyh](https://developer.apple.com/news/?id=3d8a9yyh)

This [analyzer](https://github.com/crasowas/app_store_required_privacy_manifest_analyser/tree/main) reported that patrol needs to add a privacy policy for these APIs:

```sh
🛠️  Descriptions for the following required API reason(s) may be missing: 1
[0] NSPrivacyAccessedAPICategoryFileTimestamp
```

### Proposal

https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/adding_a_privacy_manifest_to_your_app_or_third-party_sdk

### PR DESCRIPTION
**Pull request by piotruela**
Closes #2189.

Updated version of [`Telegraph`](https://github.com/Building42/Telegraph) was not released on CocoaPods so we need to stick to the vendored package. Turned out that we were not using the part of code that was accessing  `fileModificationDate` so I removed it. I also added empty privacy manifest to get rid of all the warning from this [analyzer](https://github.com/crasowas/app_store_required_privacy_manifest_analyser/tree/main).

### CODE REVIEW

File: packages/patrol/darwin/Classes/Telegraph/Protocols/HTTP/Routing/Server+Routing.swift [hunk 1]
```diff
@@ -37,33 +37,6 @@ extension Server {
   }
 }
 
-// MARK: Serve static content methods
```
Comment (mateuszwojtczak) [inline packages/patrol/darwin/Classes/Telegraph/Protocols/HTTP/Routing/Server+Routing.swift:40]: why do we remove it? is it somehow related to the privacy manifest?

**Comment by mateuszwojtczak** [comment]
apart from my question, the fix LGTM

File: packages/patrol/darwin/Classes/Telegraph/Protocols/HTTP/Routing/Server+Routing.swift (hunk 1 above)
Comment (mateuszwojtczak) [reply packages/patrol/darwin/Classes/Telegraph/Protocols/HTTP/Routing/Server+Routing.swift:40]: ok, now I get it, sorry

**Review by mateuszwojtczak — APPROVED** [review]

**Review by pdenert — APPROVED** [review]
lgtm

**Pull request merged**